### DISCUSSION TITLE
How do I edit the wiki for Quick?

### DISCUSSION THREAD

**Discussion by jiroikedafeingold** [Quick/Quick#1193] [Q&A] [answered]
I'd like to edit the Projects who use Quick to add an entry, but I don't see an edit button on the Wiki page and I can't seem to clone the wiki repro with the provided url https://github.com/Quick/Quick.wiki.git

**Comment by younata** [comment, marked as answer]
Sorry about that, I re-opened the wiki for editing.

**Comment by jiroikedafeingold** [comment]
Thanks!
